rlberry/agents/torch/dqn/dqn_utils.py

```diff
@@ -8,6 +10,38 @@
 logger = rlberry.logger
 
 
+def stable_scaled_log_softmax(x, tau, axis=-1):
+    """Scaled log_softmax operation.
+    Args:
```

Comment:
`Parameters `instead of `Args` ? the is the convention we have.

Reply:
and with the usual formatting with ----------- below the parameters and returns keywords. This is if one day we want to include this in the API and also to have somthing well formated when asking the docstring directly from python